README.md

```diff
@@ -0,0 +1,330 @@
+Maven.configureResolver().workOffline().resolve("G:A:V").withTransitivity().asFile();
+Maven.configureResolver().useLegacyLocalRepo(true).resolve("G:A:V").withTransitivity().asFile();
+```
+
+#### Specify settings.xml
+```java
+Maven.configureResolver().fromFile("/path/to/settings.xml")
+  .resolve("G:A:V").withTransitivity().asFile();
+
+Maven.configureResolver().fromClassloaderResource("path/to/settings.xml")
+  .resolve("G:A:V").withTransitivity().asFile();
+```
+
+<sub>**Warning:** ShrinkWrap Resolvers doesn't consume settings.xml specified on the command line or in the IDE. It reads settings.xml files at standard locations: `~/.m2/settings.xml` and `$M2_HOME/conf/settings.xml`, unless overridden in the API or via System property.</sub>
+
+#### Define Maven repositories manually
+
+```java
+Maven.configureResolver().withRemoteRepo("my-repository-id", "url://to/my/repository", "layout")
+  .resolve("G:A:V").withTransitivity().asFile();
+
+Maven.configureResolver().withRemoteRepo(MavenRemoteRepositories.createRemoteRepository("my-repository-id", "url://to/my/repository", "layout"))
+  .resolve("G:A:V").withTransitivity().asFile();
+```
+
+### Resolution of Artifacts from POM Files
+
+In Maven projects, you likely specify dependencies in your pom.xml file. ShrinkWrap Resolvers allows you to avoid redundancy by automatically loading metadata from there.
+
+- To load dependencies from the POM:
+  ```java
+  Maven.resolver().loadPomFromFile("/path/to/pom.xml").resolve("G:A").withTransitivity().asFile();
+  ```
+- Or, for dependencies with non-JAR packaging or classifier:
+  ```java
+  Maven.resolver().loadPomFromClassLoaderResource("/path/to/pom.xml").resolve("G:A:P:?").withTransitivity().asFile();
+  ```
+- To import and resolve dependencies based on specific scopes:
+  ```java
+  Maven.resolver().loadPomFromFile("/path/to/pom.xml").importDependencies(ScopeType.TEST, ScopeType.PROVIDED).resolve().withTransitivity().asFile();
+  ```
+- Or use convenience methods:
+  ```java
+  Maven.resolver().loadPomFromFile("/path/to/pom.xml").importRuntimeDependencies().resolve().withTransitivity().asFile();
+  ```
+
+- To activate profiles:
+  ```java
+  Maven.resolver().loadPomFromFile("/path/to/pom.xml", "activate-profile-1", "!disable-profile-2").importRuntimeAndTestDependencies().resolve().withTransitivity().asFile();
+  ```
+
+### Version Range Resolution
+The ShrinkWrap Resolver API allows for resolution of available versions info from a requested range. The [Maven documentation](http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html) specifies the version range syntax.
+```java
+final MavenVersionRangeResult versionRangeResult = Maven.resolver().resolveVersionRange("G:A:[1.0.0]");
+```
+
+### System properties
+
+ShrinkWrap Resolvers allows you to override any programmatic configuration via System properties.
+
+- `org.apache.maven.user-settings`
```

Comment:
These should probably be explained a little bit. I'm not talking about full table (although it is an option), but providing just the properties and not what they stand for is confusing